it/controllers/CalculationControllerISpec.scala

```diff
@@ -431,8 +431,7 @@ class CalculationControllerISpec extends IntegrationSpecCommonBase with FeatureS
       parsedBody.select("#main-content header p span").first.text() shouldBe "The period dates are"
       parsedBody.select("#how-penalty-is-applied").text() shouldBe "This penalty applies if VAT has not been paid for 30 days."
       parsedBody.select("#30-day-calculation").text() shouldBe "It is made up of 2 parts:"
-      parsedBody.select("#main-content > div > div > ul > li:nth-child(1)").text() shouldBe "2% of £10,000.00 (the unpaid VAT 15 days after the due date) = £200.00"
-      parsedBody.select("#main-content > div > div > ul > li:nth-child(2)").text() shouldBe "2% of £10,000.00 (the unpaid VAT 30 days after the due date) = £200.00"
+      parsedBody.select("#main-content > div > div > ul > li:nth-child(1)").first.text() shouldBe "2% of £10,000.00 (the unpaid VAT 15 days after the due date) = £200.00"
```

Comment:
There shouldn't be = £200 at the end, I think it's because the test uses penaltyDetailsWithDueDateMoreThan30daysAccruing which isn't right for estimated penalties